web/src/lib/api/api.ts

```diff
@@ -0,0 +1,25 @@
+import { get } from '$lib/req';
```

Comment:
`$lib` is actually an alias to some path. You'll find it in the closest `tsconfig.json` file. It allows us to use basically "absolute" paths. Some sigil is usually used to make it clear that we're using an alias; you can otherwise just configure it without the sigil and just have `lib/req`, but hopefully it's clear why it can look confusing: are we looking at a library?

Common sigils people use: `$`, `@`, `~`. Why `~` is common when it's `$HOME` for those who know it? Idk. Same goes for `$lib` cause that looks like a bash variable but it's less glaring cause we know `$lib/req` can't be a bash string here.